perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -192,6 +190,9 @@ export default {
     showMenu: false
   }),
   computed: {
+    canBuyProduct() {
```

Comment:
either we change the name of the computed or we change the logic of the returned value

Reply:
done